tests/Spinner.test.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
Can you please move component tests into `tests/components/Spinner.test.jsx` ? I mean ,let's try to follow file structure we have in `src/` in order to make it easier to navigate.

Reply:
sure